### ISSUE TITLE
Error when contact is blocked

### ISSUE THREAD

**Issue by dougpiv** [wppconnect-team/wppconnect#340]
## Description

When I try to use the function sendText() and the number is blocked on the client side, is just return a error message and don't let me catch the error.

## Environment

- **Wppconnect version(s): 1.8.5
- **Browser: 
- **OS: Windows 10
- **Node version: v12.18.3

## Steps to Reproduce

1. Create the client
2. try to send a text with sendText() function to a contact that is blocked on the whatsapp
3. it appears the error

## Log Output

```
info:     Checking for updates
info:     You're up to date
info:     [1:browser] Initializing browser...
ContactBlocked: Contact is blocked
    at new i (https://web.whatsapp.com/bootstrap_qr.e715964683fa904d24c6.js:2:382908)
    at new i (https://web.whatsapp.com/bootstrap_qr.e715964683fa904d24c6.js:19:144383)
    at https://web.whatsapp.com/bootstrap_qr.e715964683fa904d24c6.js:2:526064
    at t.promiseCallSync (https://web.whatsapp.com/bootstrap_qr.e715964683fa904d24c6.js:2:356476)
    at t.sendMsgRecord (https://web.whatsapp.com/bootstrap_qr.e715964683fa904d24c6.js:2:525740)
    at https://web.whatsapp.com/bootstrap_main.cc588fb013259c2e0ec4.js:2:7283
    at q (https://web.whatsapp.com/bootstrap_qr.e715964683fa904d24c6.js:2:353990)
    at MutationObserver.H (https://web.whatsapp.com/bootstrap_qr.e715964683fa904d24c6.js:2:353571)
```

## Your Code

```
wppconnect
    .create({
        session: data[0].username,
        folderNameToken: 'tokens', //folder name when saving tokens
        mkdirFolderToken: '', //folder directory tokens, just inside the venom folder, example:  { mkdirFolderToken: '/node_modules', } //will save the tokens folder in the node_modules directory
        headless: true, // Headless chrome
        devtools: false, // Open devtools by default
        useChrome: true, // If false will use Chromium instance
        debug: false, // Opens a debug session
        browserWS: '', // If u want to use browserWSEndpoint
        browserArgs: chromiumArgs,
        logQR: true, // Logs QR automatically in terminal                
        disableWelcome: true, // Will disable the welcoming message which appears in the beginning
        updatesLog: true, // Logs info updates automatically in terminal
        autoClose: 0, // Automatically closes the venom-bot only when scanning the QR code (default 60 seconds, if you want to turn it off, assign 0 or false)                
        createPathFileToken: true,
        waitForLogin: true
    })
    .then(async (client) => {
        await client.waitForLogin().then(async () => {
            try {
                await client.sendText('55' + numero + '@c.us', 'message').then((result) => {
                        console.log(result);
                    })
                    .catch((erro) => {
                        console.error('Error when sending: ', erro); //return object error
                    });
            } catch (error) {
                console.log('deu erro:', error)
            }
        }).catch((erro) => {
            console.log('Erro no waitforlogin:', erro);
        })
    })
    .catch((erro) => {
        console.log('Ta caindo nesse erro:', erro);
    });
```

**Comment by leobevi** [comment]
I didn't really understand your code, but you can do a validation of the number before submitting

wppconnect
    .create({
        session: data[0].username,
        folderNameToken: 'tokens', //folder name when saving tokens
        mkdirFolderToken: '', //folder directory tokens, just inside the venom folder, example:  { mkdirFolderToken: '/node_modules', } //will save the tokens folder in the node_modules directory
        headless: true, // Headless chrome
        devtools: false, // Open devtools by default
        useChrome: true, // If false will use Chromium instance
        debug: false, // Opens a debug session
        browserWS: '', // If u want to use browserWSEndpoint
        browserArgs: chromiumArgs,
        logQR: true, // Logs QR automatically in terminal                
        disableWelcome: true, // Will disable the welcoming message which appears in the beginning
        updatesLog: true, // Logs info updates automatically in terminal
        autoClose: 0, // Automatically closes the venom-bot only when scanning the QR code (default 60 seconds, if you want to turn it off, assign 0 or false)                
        createPathFileToken: true,
        waitForLogin: true
    })
    .then(async (client) => {
        await client.waitForLogin().then(async () => {
            try {
				let checknumber = await srv.checkNumberStatus(numero+'@c.us');  
				if(checknumber.canReceiveMessage===true){
					await client.sendText(checknumber.id._serialized, 'message').then((result) => {
							console.log(result);
						})
						.catch((erro) => {
							console.error('Error when sending: ', erro); //return object error
						});
				}
            } catch (error) {
                console.log('deu erro:', error)
            }
        }).catch((erro) => {
            console.log('Erro no waitforlogin:', erro);
        })
    })
    .catch((erro) => {
        console.log('Ta caindo nesse erro:', erro);
    });

**Comment by dougpiv** [comment]
{
  id: {
    server: 'c.us',
    user: '55xxxxxxxxxx',
    _serialized: '55xxxxxxxxxx@c.us'
  },
  status: 200,
  isBusiness: false,
  canReceiveMessage: true,
  numberExists: true
}

this is the return of the checknumber, the number that i put there is blocked, when it goes to the if verification it get in and do the error again

**Comment by dougpiv** [comment]
I "solve" the problem doing a verification:
```
var blocklist = await client.getBlockList();
```
if the number are in the list i dont send.

**Comment by edgardmessias** [comment]
Fixed at v1.8.6
